etna/transforms/encoders/categorical.py

```diff
@@ -0,0 +1,235 @@
+import warnings
+from typing import Optional
+
+import numpy as np
+import pandas as pd
+from sklearn import preprocessing
+from sklearn.utils._encode import _check_unknown
+from sklearn.utils._encode import _encode
+from sklearn.utils._encode import _unique
+
+from etna.transforms.base import PerSegmentWrapper
+from etna.transforms.base import Transform
+
+
+class LE(preprocessing.LabelEncoder):
```

Comment:
Make it `_LabelEncoder`, because this class is not intended for users and longer name makes it easier to understand class meaning.